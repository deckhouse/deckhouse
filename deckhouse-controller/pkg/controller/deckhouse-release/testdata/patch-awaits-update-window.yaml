---
apiVersion: deckhouse.io/v1alpha1
kind: DeckhouseRelease
metadata:
  name: v1.25.0
spec:
  version: "v1.25.0"
status:
  phase: Deployed
---
apiVersion: deckhouse.io/v1alpha1
kind: DeckhouseRelease
metadata:
  name: v1.25.1
spec:
  version: "v1.25.1"
status:
  phase: Pending
---
apiVersion: deckhouse.io/v1alpha1
kind: DeckhouseRelease
metadata:
  name: v1.26.0
spec:
  version: "v1.26.0"
status:
  phase: Pending
