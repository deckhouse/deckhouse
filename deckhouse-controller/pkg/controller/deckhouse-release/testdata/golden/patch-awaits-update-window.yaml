---
apiVersion: deckhouse.io/v1alpha1
approved: false
kind: DeckhouseRelease
metadata:
  creationTimestamp: null
  name: v1.25.0
  resourceVersion: "999"
spec:
  version: v1.25.0
status:
  approved: false
  message: ""
  phase: Deployed
  transitionTime: null
---
apiVersion: deckhouse.io/v1alpha1
approved: false
kind: DeckhouseRelease
metadata:
  annotations:
    release.deckhouse.io/isUpdating: "false"
    release.deckhouse.io/notified: "true"
  creationTimestamp: null
  name: v1.25.1
  resourceVersion: "1001"
spec:
  version: v1.25.1
status:
  approved: false
  message: Release is waiting for the update window until 18 Oct 19 08:00 UTC
  phase: Pending
  transitionTime: null
---
apiVersion: deckhouse.io/v1alpha1
approved: false
kind: DeckhouseRelease
metadata:
  creationTimestamp: null
  name: v1.26.0
  resourceVersion: "1000"
spec:
  version: v1.26.0
status:
  approved: false
  message: awaiting for v1.25.1 release to be deployed
  phase: Pending
  transitionTime: null
